### ISSUE TITLE
Error following Quickstart instructions: Assert failed (symbol? read-ns)

### ISSUE THREAD

**Issue by barrosfelipe** [thheller/shadow-cljs#686]
Brief description:

I get an error when trying to run a browser REPL after creating a new project with `create-cljs-project`.

A tab opens but already in the "Problem loading page: unable to connect" page.

Steps to reproduce (following quickstart instructions):

```npx create-cljs-project test-project```
```npx shadow-cljs browser-repl```

Error:
```
shadow-cljs - config: /home/user/Repositories/test-project/shadow-cljs.edn
shadow-cljs - server version: 2.8.96 running at http://localhost:9630
shadow-cljs - nREPL server started on port 42393
[:browser-repl] Configuring build.
[:browser-repl] Compiling ...
[:browser-repl] Build completed. (130 files, 0 compiled, 0 warnings, 1.94s)
null=> Execution error (AssertionError) at shadow.cljs.repl/read-one (repl.clj:535).
Assert failed: (symbol? read-ns)

Full report at:
/tmp/clojure-165698468704508128.edn
Running shutdown hook.
shutting down ...
Worker shutdown.
```

Expected behaviour:

REPL connects to browser tab and works.

System information:
shadow-cljs "2.8.96"
npm "6.14.4
node "v13.11.0"

**Comment by rigoleto** [comment]
I see the same thing with `node-repl`. Switching to v2.8.94 works for me.

**Comment by thheller** [comment]
Yeah, stay with `2.8.94` for now. I accidentally introduced a timing issue where the REPL only works when an actual runtime is connected and trying to use it before will break with the error above.

Will fix soon.

**Comment by thheller** [comment]
Fixed in `2.8.97`.
